reventlov/bot.py

```diff
@@ -60,6 +61,25 @@ def help_message(self):
             msg = '{}\n{}'.format(msg, handler_msg)
         return msg.replace('_', '\_')
 
+    @property
+    def disabled_plugins(self):
+        return ','.join(
+            [name for name in self.plugins if self.plugins[name] is None]
```

Comment:
Probably better to use `", "` as join string, for better readability.

Also, if the list is sorted, it would be easy to find if some plugin is disabled.